jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -31,6 +31,14 @@
             description: 'sha1 or refname (e.g. origin/pr/N/head) to build'
     scm:
         - git:
+            url: https://github.com/kubernetes/test-infra
```

Comment:
Ah, according to http://stackoverflow.com/questions/14843696/checkout-multiple-git-repos-into-same-jenkins-workspace it might not work.